src/app/features/room-nav/RoomNavItem.tsx

```diff
@@ -325,6 +379,34 @@ export function RoomNavItem({
               </FocusTrap>
             }
           >
+            {room.isCallRoom() && (
+              <TooltipProvider
+                position="Bottom"
+                offset={4}
+                tooltip={
+                  <Tooltip>
+                    <Text>Open Chat</Text>
+                  </Tooltip>
+                }
+              >
+                {(triggerRef) => (
+                  <IconButton
+                    ref={triggerRef}
+                    data-testid="chat-button"
+                    onClick={handleChatButtonClick}
+                    aria-pressed={isChatOpen}
```

Comment:
Button should not be pressed if the room is not selected
```suggestion
                    aria-pressed={isChatOpen && selected}
```